vllm/entrypoints/openai/api_server.py

```diff
@@ -268,11 +268,15 @@ async def show_version():
 @router.post("/v1/chat/completions")
 async def create_chat_completion(request: ChatCompletionRequest,
                                  raw_request: Request):
+
     generator = await openai_serving_chat.create_chat_completion(
         request, raw_request)
+
+    # if there's an error, return it
```

Comment:
Imo this comment (and the nearby whitespace changes) are unnecessary.

Reply:
bba73949